src/Configuration/test/Encryption.Test/Cryptography/TextDecryptorFactoryTest.cs

```diff
@@ -2,52 +2,52 @@
 // The .NET Foundation licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information.
 
-using Steeltoe.Configuration.Encryption.Decryption;
+using Steeltoe.Configuration.Encryption.Cryptography;
 
-namespace Steeltoe.Configuration.Encryption.Test.Decryption;
+namespace Steeltoe.Configuration.Encryption.Test.Cryptography;
 
-public sealed class EncryptionFactoryTest
+public sealed class TextDecryptorFactoryTest
 {
     [Fact]
-    public void Create_WhenDisabled_CreateNoopDecryptor()
+    public void Create_WhenDisabled_CreateNoneDecryptor()
     {
-        var configServerEncryptionSettings = new ConfigServerEncryptionSettings();
-        Assert.IsType<NoopDecryptor>(EncryptionFactory.CreateEncryptor(configServerEncryptionSettings));
+        var configServerEncryptionSettings = new ConfigServerDecryptionSettings();
+        Assert.IsType<NoneDecryptor>(TextDecryptorFactory.CreateDecryptor(configServerEncryptionSettings));
     }
 
     [Fact]
     public void Create_WhenEnabledWithKey_CreateAesDecryptor()
     {
-        var configServerEncryptionSettings = new ConfigServerEncryptionSettings
+        var configServerEncryptionSettings = new ConfigServerDecryptionSettings
```

Comment:
`configServerEncryptionSettings` could be renamed to `configServerDecryptionSettings` a few times in this file to keeps things fully aligned